rules/cross-platform/linux_process_started_in_temp_directory.toml

```
@@ -17,13 +17,13 @@ index = ["auditbeat-*", "logs-endpoint.events.*"]
 language = "kuery"
 license = "Elastic License"
 name = "Unusual Process Execution - Temp"
+note = "Auditbeat configurations: For macOS systems, Auditbeat must be run with an account that has access to the binaries in the /usr/sbin, /sbin, /usr/bin, and /bin directories. For Linux systems, to avoid conflicts with auditd, you will need to stop and disable auditd with 'sudo service stop auditd && sudo systemctl disable auditd'."
 risk_score = 47
 rule_id = "df959768-b0c9-4d45-988c-5606a2be8e5a"
 severity = "medium"
-tags = ["Elastic", "Linux"]
+tags = ["Elastic", "Linux", "macOS"]
 type = "query"
 
 query = '''
-event.category:process and event.type:(start or process_started) and process.working_directory:/tmp
+event.category:process and event.type:start and process.working_directory:(/tmp or /private/tmp)
```

Add `/var/tmp`, which is often the same directory as `/tmp`, but even if not, it's always world writable. Ditto for `/private/var/tmp`

```suggestion
event.category:process and event.type:start and process.working_directory:(/tmp or /var/tmp  or /private/tmp or /private/var/tmp)
```